src/ergw_config.erl

```diff
@@ -30,13 +31,16 @@
 			 {apns, []},
 			 {charging, [{default, []}]}]).
 -define(VrfDefaults, [{features, invalid}]).
--define(ApnDefaults, [{ip_pools, []}]).
+-define(ApnDefaults, [{ip_pools, []}, {bearer_type, 'IPv4v6'}, {prefered_bearer_type, 'IPv6'}]).
 -define(DefaultsNodesDefaults, [{vrfs, invalid}, {node_selection, default}]).
 
 -define(is_opts(X), (is_list(X) orelse is_map(X))).
 -define(non_empty_opts(X), ((is_list(X) andalso length(X) /= 0) orelse
 			    (is_map(X) andalso map_size(X) /= 0))).
 
+-define(IS_IPv4(X), (is_tuple(X) andalso tuple_size(X) == 4)).
+-define(IS_IPv6(X), (is_tuple(X) andalso tuple_size(X) == 8)).
+
```

Comment:
Is it necessary to have is_tuple/1 in the guard?
Or is this a way to speed up the test?

Reply:
I'm not sure, it might enough to have the tuple_size/1 guard. The same might apply to the map_size/1 guard.